domain/usecases/transfers/list_transfer.go

```diff
@@ -14,7 +14,7 @@ func (s TranfersUsecase) ListTransfer(ctx context.Context, originId string) ([]t
 		return []transfers.Transfer{}, account.ErrIdNotExists
 	}
 
-	list, err := s.repo.ListTransfer(originId)
+	list, err := s.repo.ListTransfer(context.Background(), originId)
```

Comment:
repassar o ctx recebido
```suggestion
	list, err := s.repo.ListTransfer(ctx, originId)
```